adds a note about using aggregate with pagination (#290)
neo4j/docs-graphql: modules/ROOT/pages/queries-aggregations/aggregations.adoc

}
----

> == A note about aggregate with pagination

Reviewer: ```suggestion
== Aggregations and pagination
```

> Because aggregate is done within the `Connection` type, the optional arguments `first` and `after` are available for pagination. However, pagination **does not** apply to aggregate

Reviewer: ```suggestion
Since aggregations are done within the `Connection` type, the optional arguments `first` and `after` are available for pagination.
However, pagination **does not** apply to aggregations.
```

For example:

[source, graphql, indent=0]
----
query {
    usersConnection(first: 10) {
        aggregate {
            count {
                nodes
            }
        }
    }
}
----

> The result would return the count of all the nodes:

Reviewer: ```suggestion
The result returns the count of all the nodes:
```
